node/src/blockchain/signature.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
 use serde_derive::{Deserialize, Serialize};
 
+//TODO what is this for? It seems unused.
```

Comment:
You could try deleting this and see if anything breaks.

Reply:
Yeah, I tried and it really seems to be used in serialization process, probably by things we do need to keep.